pkg/interop/native/neo/neo.go

```diff
@@ -0,0 +1,39 @@
+package neo
+
+import (
+	"github.com/nspcc-dev/neo-go/pkg/interop"
+	"github.com/nspcc-dev/neo-go/pkg/interop/contract"
+)
+
+// Hash represents NEO contract hash.
+const Hash = "\x32\xe9\x21\xac\xc4\x75\x06\xb8\xc3\xa7\xdd\xde\xbd\x1a\x9d\x68\xca\xba\x17\xf6"
+
+// Symbol represents `symbol` method of NEO native contract.
+func Symbol() string {
+	return contract.Call(nil, "symbol", contract.ReadStates).(string)
+}
+
+// Decimals represents `decimals` method of NEO native contract.
+func Decimals() int {
+	return contract.Call(interop.Hash160(Hash), "decimals", contract.ReadStates).(int)
+}
+
+// TotalSupply represents `totalSupply` method of NEO native contract.
+func TotalSupply() int {
+	return contract.Call(interop.Hash160(Hash), "totalSupply", contract.ReadStates).(int)
+}
+
+// BalanceOf represents `balanceOf` method of NEO native contract.
+func BalanceOf(addr interop.Hash160) int {
+	return contract.Call(interop.Hash160(Hash), "", contract.ReadStates, addr).(int)
+}
+
+// Transfer represents `transfer` method of NEO native contract.
+func Transfer(from, to interop.Hash160, amount int) bool {
+	return contract.Call(interop.Hash160(Hash), "transfer", contract.All, from, to, amount).(bool)
+}
+
+// GetCommittee represents `getCommittee` method of NEO native contract.
+func GetCommittee() []interop.PublicKey {
```

Comment:
Can we also include `getCandidates`, `getNextBlockValidators` and `getGasPerBlock`?

Reply:
Fixed.